shopwareerpconnect/models/partner.py

```diff
@@ -563,9 +563,11 @@ def search(self, filters=None):
 
         :rtype: list
         """
-        return [int(row['customer_address_id']) for row
-                in self._call('%s.list' % self._shopware_model,
-                              [filters] if filters else [{}])]
+        return self._call('%s.Search' % self._shopware_model,[filters] if filters else [{}])
```

Comment:
Is there a change of call  ? List does not exist and replaced by Search?
@AnukPatel